src/HotChocolate/Core/src/Execution/Pipeline/PersistedQueries/IPersistedQueryOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace HotChocolate.Execution.Pipeline
+{
+    /// <summary>
+    /// Represents common options for persisted queries
+    /// </summary>
+    public interface IPersistedQueryOptions
```

Comment:
Any particular reason why you decided to introduce this as a separate interface? Conceptually the option to only allow persisted queries would live in the [`GraphQLServerOptions`](https://github.com/ChilliCream/hotchocolate/blob/c1eb36cf3542572372847c5e38f4bc349bc22bfa/src/HotChocolate/AspNetCore/src/AspNetCore/GraphQLServerOptions.cs) for me. Here you already configure what types of requests are allowed.